target/linux/pistachio/base-files/lib/netifd/proto/lowpan.sh

```diff
@@ -19,9 +19,17 @@ proto_lowpan_setup() {
         local id=${ifname##*[[:alpha:]]}
         local interface="$cfg$id" # i.e wpan0
 
-        generate_mac $WPAN_OTP_REG0
-        local WPAN_EUI=$(echo $MAC | cut -d: -f1,2,3):ff:fe:$(echo $MAC | cut -d: -f4,5,6)
+        # Wifi mac address must be part of DTB to get access to it
```

Comment:
Can we add a check and a log message so someone doesn't spend hours debugging when they don't have one